File: 프로그래머스/lv0/120878. 유한소수 판별하기/유한소수 판별하기.java
class Solution {
    public int solution(int a, int b) {
        while(b%5 == 0) b /= 5;
        while(b%2 == 0) b /= 2;
        
        if(a%b == 0) return 1;
        return 2;
    }
} 

File: 프로그래머스/0/120878. 유한소수 판별하기/유한소수 판별하기.js
function solution(a, b) {
    while (b % 2 === 0) {
        b /= 2;
    }
    while (b % 5 === 0) {
        b /= 5;
    }
    
    if (a % b === 0) {
        return 1;
    }
    return 2;
}
